src/components/DataDrawer/DataTable/DataTableRow/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { TableRow, TableCell } from '@material-ui/core';
+
+export interface TableRowProps {
+  className?: string;
+  rowData: any;
```

Comment:
This shouldn't be `any` - based on the `map` function, we know its type should be something like `rowData: [string, string | number][]`.

Personally, I'd probably refactor it to

`rowData: { key: string, value: string | number }[]` (explicit key'd object vs. position-dependent array)

but if that requires a bunch of reformatting the data, it doesn't really matter.